presto/test_presto.py

```diff
@@ -32,7 +32,7 @@ def __create_schema_via_hive(self, name, schema):
 
     def __verify_schema_via_presto(self, name, schema):
         query = "show schemas;"
-        presto_cmd = "presto --catalog=hive --execute='{}' --output-format TSV".format(
+        presto_cmd = "presto --server=localhost:8060 --catalog=hive --execute='{}' --output-format TSV".format(
```

Comment:
Likely, the tests need no change, since default is now back to Presto default.

Reply:
Yes, for init actions we don't need to have `dataproc` prefix.

Reply:
Done.